test/scripts/examples/process-quarkus-example.feature

```diff
@@ -0,0 +1,17 @@
+@quarkus
+Feature: Build process-quarkus-example images
+
+  Background:
+    Given Clone Kogito examples into local directory
+
+  Scenario: Build process-quarkus-example images
+    Then Local example service "process-quarkus-example" is built by Maven using profile "default" and deployed to runtime registry
+    And Local example service "process-quarkus-example" is built by Maven using profile "persistence" and deployed to runtime registry
+    And Local example service "process-quarkus-example" is built by Maven using profile "events" and deployed to runtime registry
```

Comment:
do we need "only-events" example ?

Reply:
Currently no, I put it there to have all the combinations created.

Reply:
I can remove it